### PR TITLE
fix(contact-info-avatar): show avatar halo as required

### PR DESCRIPTION
**Pull request by ramya-dp**
# PR Title

show avatar halo as required

## :hammer_and_wrench: Type Of Change

- [X] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

show avatar halo in certain cases. Will be used to show the avatar when the remote speaker talks.
[Figma](https://www.figma.com/file/wsbZXvBcOlqPXZK38Cp2Od/%5BDP-67745%5D-Call-bar---Caller-audio-indicators?type=design&node-id=613-159030&mode=design&t=Io6jvvQ0FAITtxme-0)

## :bulb: Context

show avatar halo in certain cases. 
[Figma](https://www.figma.com/file/wsbZXvBcOlqPXZK38Cp2Od/%5BDP-67745%5D-Call-bar---Caller-audio-indicators?type=design&node-id=613-159030&mode=design&t=Io6jvvQ0FAITtxme-0)

## :pencil: Checklist

- [X] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs

Screenshot 2023-07-06 at 2 40 38 PM

### CODE REVIEW

File: recipes/list_items/contact_info/contact_info.vue [hunk 1]
```diff
@@ -50,6 +53,7 @@
           :initials="avatarInitials"
           :seed="avatarSeed"
           :presence="presence"
+          :style="avatarHalo"
```
Comment (braddialpad) [inline recipes/list_items/contact_info/contact_info.vue]: Is this supposed to be a prop?

File: recipes/list_items/contact_info/contact_info.vue [hunk 2]
```diff
@@ -26,7 +26,10 @@
               :overlay-icon="avatar.icon"
               :overlay-text="avatar.text"
               overlay-class="d-mn4 d-ba d-baw4 d-bc-white d-box-unset"
-              :avatar-class="['d-ba d-baw4 d-bc-white d-bar-pill', { 'd-mln24': index > 0 }]"
+              :avatar-class="['d-baw4 d-bc-white d-bar-pill d-ba d-bc-transparent', {
+                'd-mln24': index > 0,
+                'd-bc-brand': !!avatar.halo,
```
Comment (braddialpad) [inline recipes/list_items/contact_info/contact_info.vue:31]: Please add halo into the avatarList description comment so it shows up in documentation.

**Review by braddialpad — CHANGES_REQUESTED** [review]

File: recipes/list_items/contact_info/contact_info.vue (hunk 2 above)
Comment (iropolo) [reply recipes/list_items/contact_info/contact_info.vue:31]: Im wondering what is `avatar.halo`? 
Using !! indicates me it is probably a object, what it contains? do we have some info about this?

File: recipes/list_items/contact_info/contact_info.vue (hunk 1 above)
Comment (iropolo) [reply recipes/list_items/contact_info/contact_info.vue]: where is declared `avatarHalo` ? Im missing some information

Comment (ramya-dp) [reply recipes/list_items/contact_info/contact_info.vue]: removed. It's not reqd.

File: recipes/list_items/contact_info/contact_info.vue (hunk 2 above)
Comment (ramya-dp) [reply recipes/list_items/contact_info/contact_info.vue:31]: updated doc to include halo in the description. Using !! to handle in case it's undefined

**Review by iropolo — APPROVED** [review]
LGTM Thanks! 🚀

**Review by braddialpad — APPROVED** [review]
Thanks!

**Pull request merged**